Orange/widgets/utils/domaineditor.py

```diff
@@ -288,7 +291,7 @@ def get_domain(self, domain, data):
                     col_data = [orig_var.repr_val(x) if not np.isnan(x) else ""
                                 for x in self._iter_vals(col_data)]
                 elif type(orig_var) == ContinuousVariable:
-                    col_data = [str(int(x)) if cont_ints else orig_var.repr_val(x)
+                    col_data = [str(int(x)) if cont_ints() else orig_var.repr_val(x)
```

Comment:
Why did you change cont_ints to a function? It is now executed many times here...